packages/clay-list/src/ListWithItems.tsx

```diff
@@ -0,0 +1,257 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import classNames from 'classnames';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayList from './List';
+import ClaySticker from '@clayui/sticker';
+import React from 'react';
+import {ClayCheckbox} from '@clayui/form';
+import {ClayDropDownWithItems} from '@clayui/drop-down';
+
+interface IListItem {
+	/**
+	 * Description of item.
+	 */
+	description?: string;
+
+	/**
+	 * List of actions to include in dropdown.
+	 */
+	dropdownActions?: Array<any>;
```

Comment:
I think we can use the same types as `ClayDropDownWithItems` here.
```suggestion
	dropdownActions?: React.ComponentProps<typeof ClayDropDownWithItems>['items'];
```